apps/www/_blog/2022-10-21-supabase-flutter-sdk-v1-released.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: 'supabase-flutter v1 Released'
+description: We've released supabase-flutter v1. TODO write more
```

Comment:
Remove TODO and add two or three of the new things.

Reply:
Oops! Thanks for catching this one!